### PR TITLE
Dataplane mgmt ip check

### PR DESCRIPTION
**Pull request by vrragosta**
Ensure each Weka dataplane NIC has a corresponding management IP defined in each of the container's resource configuration.

### CODE REVIEW

File: scripts.d/ta/825_ha_mgmt_ip.sh [hunk 1]
```diff
@@ -19,7 +19,14 @@ for WEKA_CONTAINER in $(sudo weka local ps --output name --no-header | grep -e c
         if [[ -n ${NET_NAME} ]]; then
             if [[ $(ip -4 -j -o addr show dev ${NET_NAME} 2>/dev/null | tr -d \"\[:blank:]) =~ "local:"([0-9\.]+) ]]; then
                 NET_IP=${BASH_REMATCH[1]}
-                if [[ ! $(weka local resources -C ${WEKA_CONTAINER} --stable | grep -e ^"Management IPs") =~ ${NET_IP} ]]; then
```
Comment (jackchallen) [inline scripts.d/ta/825_ha_mgmt_ip.sh]: This is minor, and not very likely, but if
NET_IP=192.168.123.1   and Management IPs=192.168.123.11

you'll get a false match here. It's plausible someone could typo those, and that's not an easy thing to spot with ye olde eyeballs.

Comment (vrragosta) [reply scripts.d/ta/825_ha_mgmt_ip.sh]: Yeah, sorry, this should already be addressed in the second commit.

Comment (jackchallen) [reply scripts.d/ta/825_ha_mgmt_ip.sh]: Bah, Sorry, I thought i *was* looking at the most recent.

File: scripts.d/ta/825_ha_mgmt_ip.sh [hunk 2]
```diff
@@ -19,7 +19,14 @@ for WEKA_CONTAINER in $(sudo weka local ps --output name --no-header | grep -e c
         if [[ -n ${NET_NAME} ]]; then
```
[the remark this answers is not in the document]
Comment (vrragosta) [reply scripts.d/ta/825_ha_mgmt_ip.sh:24]: Yes, I think the count should match. This check has been incorporated into the most recent commit 6bb792c89020b27be1f51ea29571b8530de2132c. Note, that I think cloud environments, using VFs, would only have a single management IP. To make this distinction was a bit "messy". Please validate this part.

**Pull request merged**